.github/workflows/build.yaml

```diff
@@ -55,39 +71,50 @@ jobs:
       run: mkdir .venv
     - name: Install dependencies
       run: make setup
-    - name: Build the package
-      # We don't need to check and test the package separately because `make dist` runs
-      # those targets first and only builds the package if they succeed.
-      # Build the sdist and wheel distribution of the package and docs as a zip file.
-      run: make dist
+
     # Audit all currently installed packages for security vulnerabilities.
     - name: Audit installed packages
       run: make audit
+
+    # Build the sdist and wheel distribution of the package and docs as a zip file.
+    # We don't need to check and test the package separately because `make dist` runs
+    # those targets first and only builds the package if they succeed.
+    - name: Build the package
+      run: make dist
+
+    # Generate the requirements.txt that contains the hash digests of the dependencies and
+    # generate the SBOM using CyclonDX SBOM generator.
+    - name: Generate requirements.txt and SBOM
+      if: matrix.os == env.ARTIFACT_OS && matrix.python == env.ARTIFACT_PYTHON
+      run: make requirements sbom
+
+    # Remove the old requirements.txt file (which includes _all_ packages) and generate a
+    # new one for the package and its actual and required dependencies only.
+    - name: Prune packages and generate required requirements.txt
+      if: matrix.os == env.ARTIFACT_OS && matrix.python == env.ARTIFACT_PYTHON
+      run: |
+        rm requirements.txt
+        make prune requirements
+
+    # Find the paths to the artifact files that will be included in the release, compute
+    # the SHA digest for all the release files and encode them using Base64, and export it
+    # from this job.
     - name: Compute package hash
       if: matrix.os == env.ARTIFACT_OS && matrix.python == env.ARTIFACT_PYTHON
       id: compute-hash
       shell: bash
       run: |
         set -euo pipefail
-        # Generate the requirements.txt that contains the hash digests of the dependencies and
-        # generate the SBOM using CyclonDX SBOM generator.
-        make requirements sbom
-        # Remove the old requirements.txt file (which includes _all_ packages) and generate a
-        # new one for the package and its actual and required dependencies only.
-        rm requirements.txt
-        make prune requirements
-        # Find the paths to the files that will be included in the release.
         TARBALL_PATH=$(find dist -name "*.tar.gz")
         WHEEL_PATH=$(find dist -name "*.whl")
         REQUIREMENTS_PATH=$(find dist -name "*-requirements.txt")
         SBOM_PATH=$(find dist -name "*-sbom.json")
-        HTML_DOCS_PATH=$(find dist -name *-docs-html.zip)
-        BUILD_EPOCH_PATH=$(find dist -name *-build-epoch.txt)
-        # Compute the sha digest for all the release files and encode them using base64.
+        HTML_DOCS_PATH=$(find dist -name "*-docs-html.zip")
+        BUILD_EPOCH_PATH=$(find dist -name "*-build-epoch.txt")
```

Comment:
Consider adding `-type f` as cmd line argument for `find` to constrain the search.

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/324/commits/5f462d5b97a64f9e7a1292fcc5ff08979bbd8110.